src/components/Menu/MenuDivider.tsx

```diff
@@ -28,17 +36,25 @@ class MenuDivider extends UIComponent<ReactProps<MenuDividerProps>, any> {
 
   static defaultProps = {
     as: 'li',
+    accessibility: menuDividerBehavior as Accessibility,
   }
 
   static propTypes = {
     ...commonPropTypes.createCommon({ content: false, children: false, color: true }),
+    accessibility: PropTypes.func,
```

Comment:
I've seen another definition:
```
accessibility: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
```

Can we create a customPropType for this in separate PR? May be create a tracking issue?

Reply:
yes, we used to support object and function, now it is just a function, probably some cleanup is needed. I will create an issue.